External/docker/Dockerfile

```diff
@@ -0,0 +1,49 @@
+# This is the Dockerfile for the Enthought Edge "external" example.
+# 
+# We use "edm-rockylinux-8" as the base image.  This is a Rockylinux-8 based
+# image which includes EDM.
+#
+# EDM dependencies for the app are brought in via a .zbundle file.  This avoids
+# the need to pass your EDM token and/or a custom edm.yaml into the Dockerfile.
+#
+# We perform a two-stage build, to avoid including the .zbundle in the layers
+# of the published image.
+
+# First stage
+
+ARG BASE_IMAGE=quay.io/enthought/edm-rockylinux-8:latest
+
+FROM $BASE_IMAGE AS stage_one
+
+ARG EDGE_BUNDLE=app_environment.zbundle
+COPY $EDGE_BUNDLE /tmp/app_environment.zbundle
+
+RUN adduser app
+USER app
+WORKDIR /home/app
+
+# Create a default EDM environment using the enthought_edge bundle
+RUN edm env import -f /tmp/app_environment.zbundle edm && edm cache purge --yes
```

Comment:
you might want to delete the zbundle to keep the image smaller

Reply:
Since this is a multi-stage build, the zbundle will never make it into the final image. Feels a bit overkill, but I borrowed this straight from the adjacent native app example.